content/en/monitors/types/ci.md

```diff
@@ -136,15 +173,15 @@ In the `Notification message` section of your monitor, add text similar to the c
 
 #### Advanced alert conditions
 
-For detailed instructions on the advanced alert options (such as evaluation delay), see the [Monitor configuration][3] page.
+For detailed instructions on the advanced alert options (such as evaluation delay), see the [Monitor configuration][4] page.
 
 ### Notifications
 
-For detailed instructions on the **Configure notifications and automations** section, see the [Notifications][4] page.
+For detailed instructions on the **Configure notifications and automations** section, see the [Notifications][5] page.
 
 #### Samples and breaching values top list
 
-When a CI Test or Pipeline monitor is triggered, samples or values can be added to the notification message.
+When a CI Pipeline, CI Test, or CD Deployments monitor is triggered, samples or values can be added to the notification message.
```

Comment:
Not sure why it's flagging the Oxford comma as you are using it correctly here :)

Reply:
Maybe Vale thinks that `triggered, samples or values` is a list of things?